src/lib/http-interceptors/error.interceptor.ts

```diff
@@ -46,7 +46,10 @@ export class ErrorInterceptor implements HttpInterceptor
                 {
                     this._alertService.error(this._translation.translate('errorInterceptor.unauthorized'));
```

Comment:
Sollte das nicht auch mit in das if gepackt werden? ErrorMeldung+routeToLogin nur bei 401ern welche von unsrem Server kommen. 
Base zB macht das ErrorHandling selbst, so würden dann mehrere Fehlermeldungen angezeigt werden.